CRVResponse/singleCounter/genconfig_point.txt

```diff
@@ -13,21 +13,24 @@ double particleGun.tmin  =  800.0;
 double particleGun.tmax  =  800.0;
 
 // position
-vector<double> particleGun.point = {-4404., 2673.11, 10200.0}; //x=1000 ,l=3000
-//vector<double> particleGun.point = {-5904., 2673.11, 10200.0}; //x=1000 ,l=6000
+//vector<double> particleGun.point = {-4404., 2673.11, 10200.0}; //x=1000 ,l=3000
```

Comment:
What are these commented-out vectors?  Alternatives?  If they are obsolete just remove them.

Reply:
These are points which I use to test the PE yields for different counter lengths. I'm going to remove the obsolete ones.